### ISSUE TITLE
Old version of libigdgmm breaking hardware transcoding with iHD driver

### ISSUE THREAD

**Issue by Ulrar** [linuxserver/docker-jellyfin#107]
## Expected Behavior
Hardware transcoding with iHD driver should work

## Current Behavior
It doesn't. There's an outdated `/usr/local/lib/libigdgmm.so.11` file that takes precedence over the good `/usr/lib/jellyfin-ffmpeg/lib/libigdgmm.so.11`, which is up to date for this driver.
Removing that file fixes the issue but of course updating the image would bring the problem back.

## Steps to Reproduce
1. Get a recent intel CPU
2. Try to to hardware transcoding using vaapi on the iHD driver

## Environment
**OS: Debian buster**
**CPU architecture:** x86_64
**How docker service was installed: linuxserver image on docker hub**
A solution could be to simply not ship `/usr/local/lib/libigdgmm.so.11` with this image, or maybe just make sure it's up to date.

## Command used to create docker container (run/create/compose/screenshot)
Nothing special, pretty standard config as described on docker hub.

## logs
```
[AVHWDeviceContext @ 0x55e86c25a440] libva: dlopen of /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so failed: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so: undefined symbol: _ZN6GmmLib16GmmClientContext25CreateCustomResInfoObjectEP32GMM_RESCREATE_CUSTOM_PARAMS__REC
```

Note that this was working perfectly fine until now, so most likely this file was added recently, or it got reverted to an old version, or whatever fix you were using to force ffmpeg to use the jellyfin path to load libraries broke.

For anyone else having this issue adding `LD_LIBRARY_PATH=/usr/lib/jellyfin-ffmpeg/lib/` to the container's env variables seems to fix it, not sure if that might have side effects but it's at least a solution for now.

Thanks

**Comment by Ulrar** [comment]
This is still a problem, we are still having to manually fix it on each update

**Comment by j0nnymoe** [comment]
This was fixed with #136
